terraform/projects/prom-ec2/paas-production/main.tf

```diff
@@ -64,6 +64,18 @@ data "pass_password" "prometheus_htpasswd" {
   path = "prometheus-basic-auth-htpasswd"
 }
 
+data "pass_password" "dm_elasticsearch_metrics_password" {
+  path = "dm-elasticsearch-metrics-password"
+}
+
+data "template_file" "extra_prometheus_scrape_configs_yaml" {
+  template = file("${path.module}/extra-prometheus-scrape-configs.yml.tpl")
+
+  vars = {
+    dm_elasticsearch_metrics_password = data.pass_password.dm_elasticsearch_metrics_password.password
+  }
+}
+
```

Comment:
```suggestion
```
`data.template_file` is deprecated, we should migrate to the tf 0.12 `templatefile` function instead.  (Yes I know data.template_file is used elsewhere here, we haven't got around to fixing it)